cross/redis/PLIST

```diff
@@ -1,6 +1,5 @@
 bin:bin/redis-benchmark
 bin:bin/redis-check-aof
-bin:bin/redis-check-dump
 bin:bin/redis-cli
```

Comment:
Is there any reason not to add `bin:bin/redis-check-rdb` here?

Reply:
None, it's a mistake:)